### PR TITLE
Add example

### PR DESCRIPTION
**Pull request by IrinaSouth**

### CODE REVIEW

File: example/main.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    # Setup cors headers to allow all domains
+    # https://flask-cors.readthedocs.io/en/latest/
+    CORS(app)
+
+    # Definition of the routes. Put them into their own file. See also
+    # Flask Blueprints: http://flask.pocoo.org/docs/latest/blueprints
+
+
+    # Create FeatureHub configuration
+    edge_url = 'https://zjbisc.demo.featurehub.io/'
```
Comment (rvowles) [inline example/main.py:31]: ideally these should come from an external config file, and we should be using our sample database for a consistent url and eval key, but this is v0.1

File: example/main.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    client_eval_key = 'default/806d0fe8-2842-4d17-9e1f-1c33eedc5f31/wFk8qfmJEkLnkkJ8A8ZPTbgZPHPTJJ*heGuAGL6U8EKOUXbbRCL'
+
+    config = FeatureHubConfig(edge_url, client_eval_key)
+    config.init()
+    fh = config.repository()
```
Comment (rvowles) [inline example/main.py:36]: what is "fh" for?

Comment (rvowles) [reply example/main.py:36]: does this app not start until you have contacted the feature repository?

File: example/main.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+
+    @app.route("/")
+    def hello_world():
+        return "Hello World"
+
+    @app.route("/foo/<someId>")
+    def foo_url_arg(someId):
+        return jsonify({"echo": someId})
+
```
Comment (rvowles) [inline example/main.py:49]: typically we add a /health/liveness endpoint that says 200 if the readiness flag is True on the repository and 503 if not.

File: example/main.py (hunk 2 above)
Comment (IrinaSouth) [reply example/main.py:36]: yes I think so, is it not right?

**Review by rvowles — APPROVED** [review]

**Pull request merged**